.tekton/pipeline-service-ckcp-testing.yaml

```diff
@@ -0,0 +1,295 @@
+                #!/usr/bin/env bash
+                export KUBECONFIG=$(workspaces.kubeconfig-dir.path)/kubeconfig
+
+                kubectl -n openshift-config  get secret/pull-secret -o yaml | sed 's/openshift-config/default/' | sed 's/\.dockerconfigjson/auth.json/' | grep -v type: | kubectl apply -f -
+                kubectl -n default apply -f - <<EOF
+                apiVersion: v1
+                kind: Pod
+                metadata:
+                  name: podman
+                spec:
+                  containers:
+                  - name: podman
+                    image: registry.redhat.io/rhel8/podman:latest
+                    resources:
+                      requests:
+                        ephemeral-storage: "2Gi"
+                      limits:
+                        ephemeral-storage: "2Gi"
+                    command:
+                    - /usr/sbin/init
+                    securityContext:
+                      privileged: true
+                    volumeMounts:
+                    - name: pull-secret
+                      mountPath: "/run/user/0/containers/"
+                      readOnly: true
+                  volumes:
+                  - name: pull-secret
+                    secret:
+                      secretName: pull-secret
+                EOF
+                kubectl -n default wait pod/podman --for=condition=Ready --timeout=90s
+                kubectl -n default describe pod/podman
+            - name: install-softwares-to-podman-container
+              image: quay.io/xinjiang/kubectl:1.25
+              resources:
+                requests:
+                  memory: 500Mi
+                  cpu: 300m
+              script: |
+                #!/usr/bin/env bash
+                export KUBECONFIG=$(workspaces.kubeconfig-dir.path)/kubeconfig
+
+                kubectl -n default exec pod/podman -- sh -c "yum install -y python3-pip xz wget git make jq gcc && yum clean all"
+                kubectl -n default exec pod/podman -- sh -c "wget -c https://go.dev/dl/go1.19.1.linux-amd64.tar.gz && tar -C /usr/local -xzf go1.19.1.linux-amd64.tar.gz && rm -rf go1.19.1.linux-amd64.tar.gz"
+                kubectl -n default exec pod/podman -- sh -c "curl --location -o /usr/local/bin/argocd https://github.com/argoproj/argo-cd/releases/download/v2.4.7/argocd-linux-amd64 && chmod +x /usr/local/bin/argocd && argocd version --client"
+                kubectl -n default exec pod/podman -- sh -c "KUBECTL_VERSION=1.24.3 && curl -LO https://storage.googleapis.com/kubernetes-release/release/v\$KUBECTL_VERSION/bin/linux/amd64/kubectl && chmod +x ./kubectl && mv ./kubectl /usr/local/bin && kubectl version --client=true"
+                kubectl -n default exec pod/podman -- sh -c "wget https://github.com/mikefarah/yq/releases/download/v4.25.1/yq_linux_amd64 -O /usr/local/bin/yq && chmod +x /usr/local/bin/yq && yq --version"
+            - name: ckcp-setup
+              image: quay.io/xinjiang/kubectl:1.25
```

Comment:
Change this to the official image. If this is a custom image you've built, then please upload this to quay.io/redhat-pipeline-service organization.

Reply:
fixed. Copied the image docker.io/bitnami/kubectl:1.25 to quay.io/redhat-pipeline-service organization.